### ISSUE TITLE
epkg dependencies not installed

### ISSUE THREAD

**Issue by progfolio** [radian-software/straight.el#1028]
Possibly related: I've experienced straight not getting dependencies of other packages than magit. 

- I installed `epkg` a couple of months ago, with a `(use-package :straight t)`. It didn't install the `closql` dependency.
- This week, I updated `epkg` using `straight-pull-all`, followed by `straight-rebuild-all`. It didn't fetch `llama`, which is a new dependency of `epkg`.

_Originally posted by @fuzzbomb in https://github.com/radian-software/straight.el/issues/964#issuecomment-1360991287_

**Comment by progfolio** [comment]
It looks like the way epkg declares its dependencies is invalid:

[epkg Package-Requires metadata
](https://github.com/emacscollective/epkg/blob/cfdc4fc3c63b663fd4c1643bd3b174cb4d79287c/lisp/epkg.el#L9-L13)

Running package.el's machinery in epkg.el:

```emacs-lisp
(lm-header "Package-Requires")
```

returns: `"("`

It also trips up straight's `straight--compute-dependencies`.
I would consider this an upstream epkg bug.

**Comment by akirak** [comment]
@progfolio You have to use `lm-header-multiline`.

**Comment by tarsius** [comment]
This is how `package` does it:

```elisp
       (and-let* ((require-lines (lm-header-multiline "package-requires")))
         (package--prepare-dependencies
          (package-read-from-string (mapconcat #'identity require-lines " "))))
```

You could get away with replacing the `package-*` functions with just `read-from-string`. In this case at least.

**Comment by progfolio** [comment]
Thanks to both of you, and sorry for the noise.
I'll fix this in straight.el.

**Comment by progfolio** [comment]
This should be fixed on the "develop" branch via #1030.
Testing is appreciated.
